### PR TITLE
Strip a ton of mozilla/firefox branding & links. Update legal

### LINKED ISSUE #4: Remove all the Mozilla-specific jigs and jags
We'll want to review, and perhaps remove and/or replace the following:

- Mozilla-specific marketing (such as the banner advertising Mozilla VPN)
- Mozilla-specific links and legal notices
- Mozilla-specific functionality (such as FXA login)
- Any other Mozilla-specific content

We should pay extra attention to the legal stuff, since this instance won't be hosted by Mozilla, so a lot of that will not apply.

### PR DESCRIPTION
**Pull request by jasoncarr0**
Fixes #4

### CODE REVIEW

**Comment by cheryllium** [comment]
Jason, this PR title purports to "strip a ton" of code from the codebase, but you have added fifteen thousand lines...

EDIT: Thanks, that's better. Please add your anime collection to your global `.gitignore` for next time!

**Comment by cheryllium** [comment]
Could you also rename the branch to something a little less... risqué? Maybe you misunderstood... We want our repository to be friendly to _minors_, not _miners_.

**Comment by jasoncarr0** [comment]
@cheryllium this branch name has a long, meaningful history to it. To rename it would be to destroy that history, and risk repeating it.

File: app/ui/privacy.js [hunk 1]
```diff
@@ -98,9 +63,6 @@ module.exports = function(state, emit) {
                 </li>
               </ul>
             </li>
-            <li class="mb-4">
```
Comment (cheryllium) [inline app/ui/privacy.js]: Let's put this section back and specify that we use DigitalOcean for hosting.

Comment (jasoncarr0) [reply app/ui/privacy.js]: Fair enough, I considered that but we should also remember to add back anything else we use as well (e.g. any cloud storage provider)

**Pull request closed without merging**